newIDE/app/src/MainFrame/EditorContainers/HomePage/PlaySection/GamesPlatformFrameContext.js

```diff
@@ -0,0 +1,358 @@
+// @flow
+import * as React from 'react';
+import GamesPlatformFrame, {
+  GAMES_PLATFORM_IFRAME_ID,
+} from './GamesPlatformFrame';
+import AuthenticatedUserContext from '../../../../Profile/AuthenticatedUserContext';
+import { generateCustomAuthToken } from '../../../../Utils/GDevelopServices/User';
+import PublicProfileContext from '../../../../Profile/PublicProfileContext';
+import RouterContext from '../../../RouterContext';
+
+// If the iframe is displaying a game, it will continue playing its sound as long as the iframe
```

Comment:
Uber uber nitpicking:

```suggestion
// If the iframe is displaying a game, it will continue playing its audio as long as the iframe
```